config/requirements.txt

```diff
@@ -13,6 +14,7 @@ djangorestframework-jwt==1.11.0
 djangorestframework==3.8.2
 factory-boy==2.11.1
 gunicorn==19.7.0
+ipython==7.0.1
```

Comment:
Maybe add a `manage.py shell_plus` command to the Makefile given the introduction of `ipython`? I hadn't heard of it before.

Reply:
It's there 😃 https://github.com/chronoscio/backend/pull/50/files#diff-b67911656ef5d18c4ae36cb6741b7965R52

Reply:
Oh haha, my bad.